WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -166,7 +176,7 @@ struct OrderPaymentSection: View {
         VStack(alignment: .leading, spacing: Constants.taxesSectionVerticalSpacing) {
             taxSectionTitle
             taxLines
-            if viewModel.taxLineViewModels.isNotEmpty && viewModel.taxBasedOnSetting.isNotEmpty {
+            if viewModel.taxLineViewModels.isNotEmpty && viewModel.taxBasedOnSetting != nil {
```

Comment:
No strong preference but I find the combination of using `.renderedIf()` and declaring the variables more readable for views than `if/else`:
```
    @ViewBuilder private var taxesSection: some View {
        let taxLineNotEmpty = viewModel.taxLineViewModels.isNotEmpty
        let taxBaseNotNil = viewModel.taxBasedOnSetting != nil

        VStack(alignment: .leading, spacing: Constants.taxesSectionVerticalSpacing) {
            taxSectionTitle
            taxLines
            taxBasedOnLine
                .renderedIf(taxLineNotEmpty && taxBaseNotNil)
        }
        .padding(Constants.sectionPadding)
    }
```

Reply:
> No strong preference but I find the combination of using .renderedIf() and declaring the variables more readable for views than if/else:

Me too! Done in 0166b52e581c8bf28dd25a4ede6d817a9ee3feb2